templates/command/new/headersOnly/conanfile.py

```diff
@@ -0,0 +1,40 @@
+# Recipe autogenerated with Conan {{ conan_version }} using `conan new --template` command
+
+from conans import ConanFile, CMake, tools
+
+class {{package_name}}Conan(ConanFile):
+    name = "{{ name }}"
+    version = "{{ version }}"
+    license = "HEM internal"
+    description = "{{ description }}"
+    topics = () # Insert keywords here, eg.: "Notifier", "Observer", "Observer pattern", "Design patterns"
+    url = "{{ url }}"
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    exports_sources = "*", "!build/*", "!.*/*", "!.gitignore", "!bitbucket-pipelines.yml", "!*conanfile.py", "!.clang-tidy", "!.clang-format", ".version/*",
```

Comment:
zastanawiam się, czy nie lepiej by było napisać jawnie src, test, inc, cmake, CMakeLists.txt. Jak czegoś zapomnimy, to się nie skopiuje i nie skompiluje, to się doda. A tak będą się wszystkie nieprzewidziane rzeczy kopiować.